docs/source/model/signatures.rst

```diff
@@ -750,9 +750,11 @@ model signatures in log_model calls when signatures aren't specified.
     It is **highly recommended** to always include an input example along with your models when you log them.
 
 
-By default, if input example is a dictionary, we convert it to pandas DataFrame format when saving.
-Note that for langchain, openai, pyfunc and transformers flavors, input example could be saved without
-conversion by setting ``example_no_conversion`` to ``False``.
+.. note::
+    In latest MLflow, we do not convert dictionary input example to pandas DataFrame anymore. If the model input is json-serializable object, e.g. list or dictionary,
```

Comment:
```suggestion
    Since ..., we do not convert dictionary input example to pandas DataFrame anymore. If the model input is json-serializable object, e.g. list or dictionary,
```

Can we use `since ...` instead to clarify when it starts.